webapp/app/templates/eligibility/display_success.html

```diff
@@ -25,7 +25,7 @@ <h2>{{ _('form.eligibility.success-next-step') }}</h2>
         <p class="result-text">{{ _('form.eligibility.success-next-step-text') }}</p>
     </div>
 
-    {% if 'dependent_notes' in render_info.additional_info %}
+    {% if 'dependent_notes' in render_info.additional_info and render_info.additional_info['dependent_notes'] %}
```

Comment:
Just because you had the nice idea of setting a variable in #168. Maybe we could do the same here?